docs/source/cleanvision/issue_managers/index.rst

```diff
@@ -0,0 +1,14 @@
+Issue Managers
+==============
+
+.. automodule:: cleanvision.issue_managers
+   :autosummary:
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
+
+.. toctree::
```

Comment:
Why this list of issue types specifically?

Reply:
I didn't understand your question

Reply:
nevermind sorry I misread something!